### PR TITLE
feat(config): add support for Zooz ZEN30

### PR DESCRIPTION
**Pull request by aneisch**
Add support for ZEN30 dimmer/relay combo. Sourced from OpenZwave XML, referencing zen27 to match structure.

### CODE REVIEW

**Comment by blhoward2** [comment]
Thanks. This looks pretty good. I left you some comments throughout. Please address and let us know when it's ready.

**Comment by blhoward2** [comment]
Also, please finalize this today if you can. I'm going to pull a bunch of files for a big push (including this one) and if this isn't yet merged it's going to end up with a lot of conflicts that you'll have to resolve. That's originally why @AlCalzone and I waited to do one.

**Comment by aneisch** [comment]
> Thanks. This looks pretty good. I left you some comments throughout. Please address and let us know when it's ready.

I'm sorry to bother and waste your time, I'm not seeing any comments anywhere. Can you point me to them?

**Comment by blhoward2** [comment]
It's no problem. If you look at the file changed they should appear throughout in breaks. Or above in the conversation it should say I have pending changes.

**Comment by aneisch** [comment]
> It's no problem. If you look at the file changed they should appear throughout in breaks. Or above in the conversation it should say I have pending changes.

I don't see either. Looking at https://github.com/zwave-js/node-zwave-js/pull/1590/files I can _add_ comments, but non exist from what I can see. 

Edit: Now appearing

**Review by blhoward2 — COMMENTED** [review]
Please correct.

**Comment by blhoward2** [comment]
Try now, I was on mobile.

**Comment by blhoward2** [comment]
Thanks for helping clean these up. One other change. Sorry. This file is going to need to be split by the firmware versions as they added new parameters over time. The ozw files are often wrong in this regard. If you look there is another Zooz PR open that is doing the same thing. I'd get this one in final form, then duplicate it, remove the new values so you have the older version that matches, then adjust the firmware versions and filenames.

https://www.support.getzooz.com/kb/article/393-zen30-double-switch-change-log/

Edit: looks like it has been merged: https://github.com/zwave-js/node-zwave-js/pull/1564/files

**Comment by aneisch** [comment]
> This file is going to need to be split by the firmware versions as they added new parameters over time.

I think this should be good now. Pulled out some new params in the 1.03 file, added them back for 1.05

**Comment by blhoward2** [comment]
Thanks. Annoying, I know, but please drop the leading zero on the minor number. It should be 1.3 not 1.03. It's going to fail lint otherwise.

**Comment by aneisch** [comment]
Ah, sorry about that. Fixed.

File: packages/config/config/devices/0x027a/zen30_1.5-1.5.json
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zen30_1.5-1.5.json]: Min should be 0

File: packages/config/config/devices/0x027a/zen30_1.5-1.5.json
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zen30_1.5-1.5.json]: Capitalize

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Just a few more changes and then I think it's ready for @AlCalzone. Thanks!

**Comment by aneisch** [comment]
Once more.

**Comment by blhoward2** [comment]
Thanks. If you're done the bot can fix the lint issues for you (usually).

@AlCalzone will give it a final review and merge.

**Comment by aneisch** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
I'm not sure why the bot isn't working. @AlCalzone ?

**Review by blhoward2 — DISMISSED** [review]

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
Did you check to allow maintainers to edit the PR? If not the bot can't fix the lint issues.

**Comment by aneisch** [comment]
> Did you check to allow maintainers to edit the PR? If not the bot can't fix the lint issues.

Yes, it's checked.

**Comment by aneisch** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
It's not reading the command or it would comment. @AlCalzone will have to take a look.

**Comment by AlCalzone** [comment]
Shut up, bot! 😬
FYI it is disabled temporarily for everyone but me.

@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks! Will merge once the tests are done

**Pull request merged**
